lib/pco_api/actions.ex

```diff
@@ -20,14 +16,41 @@ defmodule PcoApi.Actions do
         end
       end
 
-      def process_url(url), do: api_endpoint <> url
+      def get, do: get("")
+      def get(id) when is_integer(id), do: get(Integer.to_string(id))
+      def get(url) when is_binary(url), do: request(:get, url, []) |> new
+      def get(params) when is_list(params), do: get(params, "")
+      def get(params, url) when is_list(params), do: request(:get, url, params) |> new
+
+      def get_list([%PcoApi.Record{} | rest] = records), do: do_get_list(records)
```

Comment:
I know you said it already, but using `get_list` feels kind of weird. No suggestions though so whatevs for now. 👌